spec/pal/result_set_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'spec_helper'
+require 'bolt_spec/files'
+require 'bolt_spec/pal'
+
+require 'bolt/pal'
+# TODO: clean this up
```

Comment:
What's this TODO?

Reply:
the requires. This should get cleaned up with the PAL work